commands/operator-sdk/cmd/new.go

```diff
@@ -94,6 +101,35 @@ func mustBeNewProject() {
 	}
 }
 
+func doScaffold() {
+	// create cmd/manager dir
+	fullProjectPath := filepath.Join(mustGetwd(), projectName)
+	cmdDir := filepath.Join(fullProjectPath, "cmd", "manager")
+	if err := os.MkdirAll(cmdDir, defaultDirFileMode); err != nil {
+		log.Fatalf("failed to create %v: %v", cmdDir, err)
+	}
+
+	// generate cmd/manager/main.go
+	cmdf := filepath.Join(cmdDir, "main.go")
```

Comment:
rename `cmdf` to `path` or `filePath`. That's more clear than `cmdf`.

Reply:
What about `cmdFilePath` to be exact? Because later we are going to have many *FilePath variables defined.